cmd/kubeadm/app/cmd/reset.go

```diff
@@ -295,3 +311,19 @@ func resetConfigDir(configPathDir, pkiPathDir string) {
 		}
 	}
 }
+
+//Checks if is control plane
```

Comment:
Missing space before 'Checks'. Also this comment is not adding any additional information beyond the name of the function. It is no different from just leaving the comment out. Should make it more descriptive english or drop this comment.